CONTENT/program.markdown

```diff
@@ -3,9 +3,9 @@
 [](#english)
 # Wizardry & Magic Spells
 
-## All aspects of magic interesting to a fresh wizard.
+## Everything you need to know about our Magic sessions
```

Comment:
Haha, that was on the site before, you changed it and I just changed it back :dart: